egs/mgb5/s5/local/nnet3/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# cat exp/nnet3/tdnn_1a/decode_dev/scoring_kaldi/best_wer
```

Comment:
please add your name, source eg and update the WERs

Reply:
done